src/view/screens/Settings/index.tsx

```diff
@@ -494,107 +494,107 @@ export function SettingsScreen({}: Props) {
           </Text>
         </TouchableOpacity>
         <TouchableOpacity
-          testID="preferencesHomeFeedButton"
+          testID="languageSettingsBtn"
```

Comment:
Changes in this file are just reorders.

Proposed new order:

<img width="664" alt="Screenshot 2024-04-24 at 21 26 52" src="https://github.com/bluesky-social/social-app/assets/810438/202721ee-e60f-4c80-8b7a-0b94977e379f">